modules/apps/jaggery/emm/src/emm/modules/backend-service-invoker.js

```diff
@@ -53,18 +53,21 @@ var backendServiceInvoker = function () {
      * @param successCallback a function to be called if the respond if successful.
      * @param errorCallback a function to be called if en error is reserved.
      */
-    function initiateXMLHTTPRequest(method, url, payload, successCallback, errorCallback) {
+    privateMethods.initiateXMLHTTPRequest = function (method, url, successCallback, errorCallback, payload) {
         var execute = function (count) {
```

Comment:
It's inappropriate to define a function with in another function. better moving execute() to outside if it's actually required to be declared as a separate function.